app/components/post_draft/post_draft.js

```diff
@@ -73,6 +75,7 @@ export default class PostDraft extends PureComponent {
         if (channelIsArchived || deactivatedChannel) {
             return (
                 <Archived
+                    testID={testID + '.archived'}
```

Comment:
extrapolation here and elsewhere in this file?

Reply:
Updated